### PR TITLE
Add write vertices num to FragmentWriter to adapt the latest GraphAr format

### PR DESCRIPTION
**Pull request by acezen**
Signed-off-by: acezen <qiaozi.zwb@alibaba-inc.com>

Committed-by: acezen from Dev container

What do these changes do?
-------------------------
- add write vertices num to FragmentWriter;
- Some LOG revise

Related issue number
--------------------

Fixes #issue number

### CODE REVIEW

File: modules/graph/fragment/gar_fragment_builder_impl.h [hunk 1]
```diff
@@ -120,7 +117,8 @@ boost::leaf::result<void> generate_csr_for_reused_edge_label(
       },
       concurrency);
 
-  VLOG(100) << "Finish building the CSR ..." << get_rss_pretty()
+  VLOG(100) << "[frag-" << this->fid_
+            << "] RSS after building the CSR ..." << get_rss_pretty()
```
Comment (sighingnow) [inline modules/graph/fragment/gar_fragment_builder_impl.h:121]: Could you  help to unify the similar logs in `arrow_fragment_builder_impl.h` as well?

**Review by sighingnow — APPROVED** [review]
LGTM.

File: modules/graph/fragment/gar_fragment_builder_impl.h (hunk 1 above)
Comment (acezen) [reply modules/graph/fragment/gar_fragment_builder_impl.h:121]: OK

**Comment by sighingnow** [comment]
Hi @acezen, try to rebase again main. The format and lint issue has been resolved by #1394.

**Pull request merged**